organism_name_update.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+
+""" Determine accession numbers in Organism table with missing names. 
+Fetch GenBank information for these accession numbers.
+Fill in missing names from GenBank information.
+
+Adapted from bac_info.py and acc2gb.py
+
+USAGE:
+python organism_name_update.py <email> 
+
+where:
+<email> is the email address associated with your NCBI account
+
+DEPENDENCIES:
+Biopython
+"""
+
+import sys
+from Bio import Entrez
+from Bio.SeqIO import parse
+import sqlite3
+
+sqlite_file = 'crispr.sqlite'
+
+
+conn = sqlite3.connect(sqlite_file)
+c = conn.cursor()
+
+
+#define email for entrez login
+db           = "nuccore"
+Entrez.email = sys.argv[1]
+
+
+
+def missing_names(sqlite_file):
+	'''
+	Check for and return accession numbers in Organism table that don't have names
+	'''
+	c.execute("SELECT  Accession FROM Organism WHERE OrganismName IS NULL")
+	id_list = c.fetchall()
+	return id_list
+
+
+def fetch_names(id_list):
+    organism_names = {}
+
+    # Doing 100 by 100 to make sure requests to NCBI are not too big
+    for i in range(0, len(id_list), 100):
+        j = i + 100
+        if (j >= len(id_list)):
+            j = len(id_list)
+
+        sys.stderr.write("Fetching entries from %s to %s from GenBank\n" % (i, j))
+        sys.stderr.flush()
+        result_handle = Entrez.efetch(db=db, rettype="gb", id=id_list[i:j])
```

Comment:
From what I've read, I believe it's because missing_names() returns a list of the missing accession numbers as individual tuples, like [(u'one',), (u'two',), (u'three',)] or sommething like that. I didn't have a filled in database to test on or I would have picked up on this, sorry about that! Would you be able to run the program in python and then print what id_list looks like to confirm?